percy/util/ignoreRegion.js

```diff
@@ -1,31 +1,6 @@
-class IgnoreRegion {
-  constructor(top, bottom, left, right) {
-    if (top < 0 || bottom < 0 || left < 0 || right < 0) {
-      throw new Error('Only Positive integer is allowed!');
-    }
+const { Region } = require('./region');
 
-    if (top >= bottom || left >= right) {
-      throw new Error('Invalid ignore region parameters!');
-    }
-
-    this.top = top;
-    this.bottom = bottom;
-    this.left = left;
-    this.right = right;
-  }
-
-  isValid(screenHeight, screenWidth) {
-    if (
-      this.top >= screenHeight ||
-      this.bottom > screenHeight ||
-      this.left >= screenWidth ||
-      this.right > screenWidth) {
-      return false;
-    }
-
-    return true;
-  }
-}
+class IgnoreRegion extends Region {}
```

Comment:
Is this class required now ?
I see that `IgnoreRegion` class was only used in spec to mock custom locations.